Tests/com/drew/metadata/exif/NikonType2MakernoteTest2.java

```diff
@@ -56,6 +61,30 @@ public void setUp() throws Exception
         assertNotNull(_exifSubIFDDirectory);
     }
 
+
+    @Test
+    public void testNikonPictureControl1Directory() throws Exception {
```

Comment:
We don't add images to this repo. We have a separate repo for regression testing with gigabytes of images. Please remove these tests. I can run the regression tests if you don't want to go down that rabbit hole.

Reply:
Done @drewnoakes